woocommerce/woocommerce-android
# Remove ipp feedback banner

### Description


This PR removes the code and related files used for the IPP Feedback banner in the order list screen

### Testing instructions


Check the order list screen and make sure the IPP Feedback banner is not showing anymore. 

| Before | After |
| ---- | ---- |
| <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/ac6228e0-138a-41cf-abbc-abb82784071d" width="350"/> | <img src="https://github.com/woocommerce/woocommerce-android/assets/30724184/db1c96ba-6933-49d5-835f-f86c6fa330a3" width="350"/> |
